### PR TITLE
Free sports realted APIs

### PR DESCRIPTION
**Pull request by abhijeet007rocks8**
APIs added are -:
- [Free NBA](https://www.balldontlie.io/#introduction)
- [API Football](https://www.api-football.com/documentation-v3)

Related to #2 @TheLearneer

### CODE REVIEW

File: src/content/apis/sports/api-football.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: API Football
+description:  API to access all API endpoints, which can get information about Football Leagues & Cups
+categories:
+  - sports
+features:
+  https: true
+url: https://www.api-football.com/documentation-v3
+logo: https://www.api-football.com/public/img/home1/hero-banner.png
```
Comment (TheLearneer) [inline src/content/apis/sports/api-football.md:9]: The logo has to be downloaded and place inside the repository itself.
Please check the contributing guides for more details.

File: src/content/apis/sports/free-nba.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: Free NBA
+description: Unofficial NBA API for Historical NBA Data
+categories:
+  - sports
+features:
+  https: true
+url: https://www.balldontlie.io/#introduction
+logo: https://www.balldontlie.io/images/cryingjordan.jpeg
```
Comment (TheLearneer) [inline src/content/apis/sports/free-nba.md:9]: The logo has to be downloaded and place inside the repository itself.
Please check the contributing guides for more details.

**Review by TheLearneer — CHANGES_REQUESTED** [review]
Thank you @abhijeet007rocks8 for your contributions.
For the PR to be accepted, please go through the requested change.

**Comment by abhijeet007rocks8** [comment]
@TheLearneer Requested Changes have been made

File: src/content/apis/sports/api-football.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+name: API Football
+description:  API to access all API endpoints, which can get information about Football Leagues & Cups
+categories:
+  - sports
+features:
+  https: true
+url: https://www.api-football.com/documentation-v3
+logo: API_Football.png
```
Comment (TheLearneer) [inline src/content/apis/sports/api-football.md:9]:
```suggestion
logo: API-Football.png

```

**Review by TheLearneer — CHANGES_REQUESTED** [review]

**Comment by abhijeet007rocks8** [comment]
@TheLearneer Logo Name is fixed

**Pull request merged**